charms/layers/neo4j/icon.svg

```diff
@@ -1,279 +1,287 @@
pFh+k1sc87TQ/SETXi7Kqmskbwe2YOOnoLJjbpyN5lu UHdYEgLPo4eKb0yye4y8RE/IhIcJObV9reVrsh+fRUH2oJVguMdeikywqXS+PiVqma3QJruJBoGf G84iY9slTZVhgcBblkkO4NjtpU/6Ldl+ICkSh/wA+QheK8nw3gSpRY8QAzKQXGyi6/21Yks8wEJi fUVkuSGdlaBoRTkvfTJg+RryCrwWmYCwfvPV9RJboUZeIc21X5b3sEHoZ6cXBYxF/We2ci2e7FQs 60Dt+5szL1dtjl5aOwVpHAHsCPjzIQKWZrdM71+DBM+x1wh4RGay2wXfki4tFRGoyENtvQmzt7pb bAioyAIperfowTf5iK7f0O6o/HW+YQSyw3ARHFyc6X0WSwcmvFufO2iPdUEmsQyIVrCSFkWPzF/H wmFz2LGAoeaE2v46H6BE842MYEImv1w+ol/310NNSuAjx1jA5Je7M+HZkly3gbKgx2MJH6+VlBiD w9kluk72YSWk4BzMTTYAhwIVLGCCO/AnEn3rVtb9mUMB01oTTIutyOiXy0t0nezYVQP7SLu4stOs n5d7hdnJ1ouqI08+SEbdeDJFNSD3xiTHKWWT6Dq6lQCRePjm7FEP1DyLKa5BwW+O9VTdzF/3q+YJ KXqvLEkx6SO6bkKhJ5evXXbX/ctIzSNS9XqR8f5J7+2lzXz3BCBMu+asKnqgYzdWzRF8IzX3D6QJ 19pEbc0xlbRua9u1vPrXp5KoYEsNyeUmuk72kqHsrsmOABx7njs5+z6xNQCwSbInFW7mzIG0aGRR qyrNnCaD4pkESD6cJpIDuVS8isHD6DWHare6U1/4WvPnP9xOWXABsW31Tq2jrAmQ00tZKT+tFsRO sCxVyOaO2VZ0H8rOKg7Ogonk4ag6C6856WZHGfMikhPRA5EdwSLWbCffPBxgw2QbZSY5JjqgT15I 63vUkKpXc53sJXuiWy17NmJMCAa2PkAzxRcvlI3kY2l+fxpS94pqkH3DiusJHW7mkBPcgx3hbFpP RHIietRkN8/Zi06Lj9Q8bKJb72d7MvPIvQBrpDMLJE8v0U2y64PutDeSL6UkNQ8XvIXUcccikZ9u WdHTWktZeX8aUv8K9QBLgfcZSdHDB3uCIbDpXlQkaegYJm7MxKtUzbN/uni1Xf1oNyl6dMA97byr a/5znHAI1pNd+P7rpOgBseL3FzTxpiYhfKKz6BDHT4c/XsgqybOj6DpayGyP2k8XcshCySB5Kcvv TQMtT0JUis7nLSRiqmcs6EaKzig6me1RKbow99U01SfoXcmeorc4KQ8hVQC5W4nk2VV0QrpBKk6K Tkg5hknFSdEJ6QWCbL0yDlQgokdr2mlh4NtvWUrvvPzvZW9W8tTJdPe++2tYcvNd9M7LS/ABw0wn kpOiE5LAqiXtls/LF0KdrDKg6CmsFbrTRPR6aqCheXEbvfMxIIQRSnjPoNwjsg9QIKLHB3RG2IoP brlpEb3zkSj6GitL/Re0VDRyk4IT0f0IDG9m8l1RCMHAb6A+mjyWDRN9gghORA+F6Ii8HztPCyBM NC++xy/JYZ6P0zEZET0wcrlccW5uzmJmUhfY6Ez30rm6ee8ljdx6i2VSbyJ6qMDi6uDVhxAGya0R 92l7RTfJPUHBNSJ6lCiyRIf5TgMcwldz7UbPVth7foDITUSPE4cUpp0UVOi7K2S+hwG2hZSKiuqj 63PsySwnoiek6PPoWNpFfnoIgGXEuUITVGAiFjJVvZbL5SqKXq88r0J0ph4c2DBtLCcCET1RHKhh chJ84OHmzfyXSM2J6InDsgjZcb8E74DJzpntY4blRCCiJ26+j7ELlT8aIrjHpraqjXKc7goRXRSM k6oHB4JwXSssZnsZiUl0Z4jooqg6FmOJ9dOpGYV3PHHvdv5LA3RXiOiiYYT9pNC+l1ZDMDW3nGgQ iOiiqDr89LL5OTK7yFcPpOYjFIQjoosKyyyubaufpRXhAtgQed9c0TuxEojoQqo6TM2i+Tki8DZK ReBgsyEOkJoT0aVSdRCdKttqm+zc/SkabhCBiC60qiP6bokWP9Oxh1aGg8luY/H00p0hossC+Jcl 1oQnsluBmoDnHuizM9lLdHeI6LKouj63iwGCTTY53JnF3nX7+AKgknrf+unOENFlNOEtZmihfQ8d uRmuDOeXY2OkdEIiurRkhwk/xn4N5mqWg3PbVu/kj9KAbvVelWnFENFlRi/rr8NchdmaRbLDdbGp AyhQPruEIka3oBpzc3MYxnhQMfrLAVd+vKTsO7JXOX3xq0zcA0TXbSLsKEEt0AohoqeJ7B0G2ZtY sr/0yUDqBz/AJ7cx14nkRPTskB0YnR5MZa85uCkIQNp03SGSE9FTT/YW9WE/a8YDk7PvKW/OvKKp fBpg5g7YxCKI5ET07PrsAPz110qD0vvtCLjBH7dplDlAZ+VE9CySfUi9evj/e+fLN7RLNqCmHHX4 /PAFRT8n76UcdiJ6lgm/2yC8Bed/OKupuwyBOig3VByXjYrjaLFAqa1EdCK7HqQb5U154Nj5zzR1 F5XwOBuHmb7kZtvWWcMKlZwS0QlVhIf/2mf3fyIRHqrdtWKTsqntSSeClxVKhCGiE+qqO0z5vN3/ w6R///jbyuTs+7FH6BFBRztmTFKpMZEGJbrDpOJEdII7wucNc77F6XumvpnUFL50djKyKa4owsE5 OC4H9TYxZpjpZXr3iOgE74TvUR92OCk8q/Qg/akLXymnLx7Xjui8Kr453NAcQtG8uM3NLDkiOBGd ELLCg/A9Xn4Ovr0Jlvwgs0liELyOWtv54ONkohMI0RG+CSqvXvvn4sUJ9Ro1NhwCKTohTtIbJv0G RT+aC5OEUOqioo8sLtI5OIGILhb5OwzSt6jXSsUazMtzRGbJa84gB7nL5HMTCAQCgUAgpB3/L8AA aAX3DDA5SvsAAAAASUVORK5CYII= "
```

Comment:
It seems that this is a png image embedded in an svg image. This will scale badly. You can find svg versions of the neo4j logo at https://neo4j.com/style-guide/